Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -39,14 +39,20 @@ function Get-AzureUri
 function Get-ProxyUri
```

Comment:
DeployRG task patch has to be updated, so that the proxy changes are picked post latest update?